gateway/ln-gateway/src/bin/gatewayd.rs

```diff
@@ -0,0 +1,97 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+use fedimint_api::{
+    core::{
+        LEGACY_HARDCODED_INSTANCE_ID_LN, LEGACY_HARDCODED_INSTANCE_ID_MINT,
+        LEGACY_HARDCODED_INSTANCE_ID_WALLET,
+    },
+    module::registry::ModuleDecoderRegistry,
+    task::TaskGroup,
+};
+use fedimint_server::{
+    config::load_from_file,
+    modules::{
+        ln::common::LightningDecoder, mint::common::MintDecoder, wallet::common::WalletDecoder,
+    },
+};
+use ln_gateway::{
+    client::{DynGatewayClientBuilder, RocksDbFactory, StandardGatewayClientBuilder},
+    config::GatewayConfig,
+    gateway::Gateway,
+    rpc::lnrpc_client::{DynLnRpcClientFactory, NetworkLnRpcClientFactory},
+};
+use tracing::{error, info};
+
+#[derive(Parser)]
+pub struct GatewayOpts {
+    /// Path to folder containing gateway config and data files
+    #[arg(long = "cfg-dir", env = "GW_CONFIG_DIR")]
+    pub cfg_dir: PathBuf,
```

Comment:
maybe call this `data_dir` as well. makes it consistent with `fedimintd` [here](https://github.com/fedimint/fedimint/blob/master/fedimintd/src/bin/main.rs#L27)